components/StatusAlert.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <p>
+    <b-alert :show="isShown" dismissible :variant="alertVariant">
+      {{ alertMessage }}
+    </b-alert>
+  </p>
+</template>
+
+<script>
+import statuses from '@/modules/statuses'
```

Comment:
Agreed the TODO is out of date, but I think there is room for improvement so I've updated the TODO to match the upcoming DPL-561 rather than remove the TODO entirely.